### ISSUE TITLE
Link Violations to DOJ Opinion Letters

### ISSUE THREAD

**Issue by bbertucc** [EqualifyApp/scan#9]
We could present the scan results in a way that we can kind of claim to be opinion letter compliant. So here don't just have a violation Of WCAG, we say "per opinion letter XXXXX, …". 

Then we can claim our scanner checks for compliance with opinion Letter XXXXX.

Relevant Link to an opinion letter: https://www.justice.gov/crt/case-document/file/1553506/download

**Comment by mgifford** [comment]
Would this be a selling point for clients?

**Comment by bbertucc** [comment]
For what it's worth, a lot of cases are up at https://www2.ed.gov/about/offices/list/ocr/frontpage/faq/readingroom.html, navigate to “case resolutions” and use “website” as a search term. 

If you look at the 2022 cases, almost all of them are regarding the accessibility of public-facing websites. The resolution agreements each have an “Appendix A” that might be helpful to you. All of these are voluntary resolution agreements that were entered into prior to making a finding of discrimination, though.

**Comment by bbertucc** [comment]
I've started to look through OCR agreements with schools that outline what they really care about. We could possibly use that data also to provide some data behind the legal weight of requirements. IE- alt text and keyboard-based navigation are frequently discussed, where color contrast is discussed a lot less.
